testsuite/features/reposync/srv_sync_products.feature

```diff
@@ -70,6 +70,7 @@ Feature: Synchronize products in the products page of the Setup Wizard
     When I open the sub-list of the product "Basesystem Module 15 SP2 x86_64"
     Then I should see that the "Server Applications Module 15 SP2 x86_64" product is "recommended"
     When I select "SUSE Linux Enterprise Server 15 SP2 x86_64" as a product
+    And I deselect "SUSE Manager Tools 15 x86_64 (BETA)" as a SUSE Manager product
```

Comment:
"as a" does not seem right English-wise... "if it is a" maybe ?

Reply:
ok, but not for this PR please, as then it will need a change in multiple lines as the step called from this step already existed and it is used in multiple scenarios as you can see.
I would prefer a more surgical change on that PR.

Reply:
you're the boss on this one :-)